infra/feast-operator/test/e2e_rhoai/resources/permissions.py

```
@@ -0,0 +1,19 @@
+from feast.feast_object import ALL_FEATURE_VIEW_TYPES
+from feast.permissions.permission import Permission
+from feast.permissions.action import READ, AuthzedAction
+from feast.permissions.policy import NamespaceBasedPolicy
+from feast.project import Project
+from feast.entity import Entity
+from feast.feature_service import FeatureService
+from feast.saved_dataset import SavedDataset
+
+perm_namespace = ["test-ns-feast"]
+
+WITHOUT_DATA_SOURCE = [Project, Entity, FeatureService, SavedDataset] + ALL_FEATURE_VIEW_TYPES
```

Ruff (0.14.10)

Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)